From 1ff36f4030287f1c263eb344d84cd4a43b1546d2 Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@math.unl.edu>
Date: Thu, 19 Nov 2015 13:23:16 -0600
Subject: [PATCH 11/11] use _DEFAULT_SOURCE instead of _BSD_SOURCE

the former is used in other kde software in general, and
the latter is generally deprecated on modern glibc's
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dbce082..051ae6d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -115,7 +115,7 @@ endif (WIN32)
 
 if (CMAKE_COMPILER_IS_GNUCXX)
    if (CMAKE_SYSTEM_NAME MATCHES Linux)
-     add_definitions (-D_BSD_SOURCE)
+     add_definitions (-D_DEFAULT_SOURCE)
      # on arm -Wcast-align throws many internal qt warning
      if(NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^arm")
        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wcast-align")
-- 
2.5.0

